src/components/Flashcard/Flashcard.js

```diff
@@ -1,37 +1,107 @@
-import React, { useState }  from 'react'
-import styles from './Flashcard.module.css'
-import { FaQuestion, FaCommentDots, FaChevronCircleLeft, FaChevronCircleRight, FaSave, FaCheck, FaTimes } from 'react-icons/fa'
+import React, { useState }  from 'react';
+import styles from './Flashcard.module.css';
+import { FaChevronCircleLeft, FaChevronCircleRight, FaUndo } from 'react-icons/fa';
 
 import { Button } from '../';
+import { CardContent } from './CardContent';
 
-export function Flashcard(props) {
-    const { flashcards } = props
-
-    const [count, setCount] = useState(0);
-        return (
-            <div className={styles.flashcardView}>
-                {/* <p>{flashcards[count].id}</p> */}
-                <div className={styles.flashcardPlusButtons}>
-                    { (count) ?  <Button onClick={() => setCount(count - 1)} className={'iconButton'}><i><FaChevronCircleLeft/></i></Button> : <p> </p>}
-                    <div className={styles.flashcard}>
-                        <div className={styles.flashcardInner}>
-                            <div className={styles.flashcardFront}>
-                                <i><FaQuestion /></i>
-                                <p>{flashcards[count].question}</p>
-                            </div>
-                            <div className={styles.flashcardBack}>
-                                <i><FaCommentDots /></i>
-                                <p>{flashcards[count].answer}</p>
-                            </div>
-                        </div>
-                    </div>
-                    { (count+1 < flashcards.length) ? <Button onClick={() => setCount(count + 1)} className={'iconButton'}><i><FaChevronCircleRight/></i></Button> : <p> </p>}
-                </div>
-                <div className={styles.flashcardUserButtons}>
-                    <Button className={'iconButton'}><i><FaSave /></i></Button>
-                    <Button className={'iconButton'}><i><FaCheck /></i></Button>
-                    <Button className={'iconButton'}><i><FaTimes /></i></Button>
-                </div>
+export function Flashcard({questions, all, known, setKnown, unknown, setUnknown, checkAllWithUserBase, setShowIntro}) {
+    const [countQuestions, setCountQuestions] = useState(1);
+    const [currentQuestion, setCurrentQuestion] = useState(questions[0]);
+    const [previousQuestions, setPreviousQuestions] = useState([questions[0].id]);
+
+    console.log(all, 'All-(known+unknown)');
```

Comment:
try to change name of variable all to for example notIncluded as it doesn't really show all of them, but just the remaining